floris/turbine_library/nrel_5MW.yaml

```diff
@@ -1,25 +1,24 @@
-
+# NREL 5MW reference wind turbine.
+# Data based on:
+# TODO: update URL
```

Comment:
Do we have the correct URL?

Reply:
The plan is for this to point here: https://github.com/NREL/turbine-models/blob/master/Offshore/NREL_5MW_126_RWT.csv but at the moment, the data hosted at that location is not intuitive (in particular, the thrust coefficient listed does not represent aerodynamic thrust). The plan is to update the data hosted at that location, and I think the URL will stay the same, but I'm hesitant to put it in when it points to unmatched data. 

How do you feel about leaving this as a TODO item and possibly creating an issue to remind us to add the URL once the data on https://github.com/NREL/turbine-models/ is updated?

Reply:
That sounds like a good plan to me

Reply:
The updated curves have now been provided in the turbine_models repo---thanks @pduff-code!! I've updated the URL.